src/model/services/controlplane/base.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpClient } from 'model/clients';
+import FetchAdapter from 'model/clients/FetchAdapter';
+
+type Configuration = import('giantswarm-cp-client').Configuration;
+
+/**
+ * Get the base configuration for the Control Plane API requests.
+ * @param client - The HTTP client to use.
+ */
+export async function getBaseConfiguration(
+  client: HttpClient
+): Promise<Configuration> {
+  const { Configuration } = await import('giantswarm-cp-client');
+  const fetchAdapter = new FetchAdapter(client);
+
+  const configBuilder = new Configuration({
+    basePath: client.requestConfig.baseURL,
+    fetchApi: fetchAdapter.fetch.bind(fetchAdapter),
+  });
+
+  return configBuilder;
```

Comment:
This creates the basic configuration that we'll use in all CP API services: 
- setting the Base URL to the CP API URL;
- creating a fetch adapter, to use our internal `HttpClient` objects.